include/z64actor.h

```diff
@@ -527,6 +527,16 @@ typedef struct DoorActorBase {
     /* 0x0000 */ DOOR_ACTOR_BASE;
 } DoorActorBase;
 
+// DoorShutter and DoorGerudo share isActive
+// Due to alignment, a substruct cannot be used in the structs of these actors.
+#define SLIDING_DOOR_ACTOR_BASE      \
+    /* 0x0000 */ DynaPolyActor dyna; \
+    /* 0x0164 */ s16 isActive // Set to true by player when using the door
```

Comment:
`// Set to true by player when using the door. Also as a timer for other niche cases in DoorShutter`
its probably worth a mention, but dont think extensive details are needed. if one is curious they can just search it up